### ISSUE TITLE
Screen reader in mode scan-off doesnt include all info displayed

### ISSUE THREAD

**Issue by elopezanaya** [microsoft/BotFramework-WebChat#5617]
### What is the PWD impact?

This is a mirror bug created for LCW, I was able to reproduce using https://microsoft.github.io/BotFramework-WebChat/01.getting-started/a.full-bundle/

When user navigates across messages , with scren reader on and scan mode off,   the narrator does not mention the information in webchat__stacked-layout__status 

for the case attached the text missing as part of the narration is "28 miinutes ago"

<img width="298" height="100" alt="Image" src="https://github.com/user-attachments/assets/61bc0c88-e96f-4117-9422-0645e96a13aa" />

User Impact:
Users relying on Screen Readers cannot identify who sent the message or when it was sent.

MAS Reference:
[MAS 4.1.2 – Name, Role, Value](https://aka.ms/MAS4.1.2)

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Transcript navigation

### How to reproduce the issue?

- screen reader on
- scan mode off
- go to https://microsoft.github.io/BotFramework-WebChat/01.getting-started/a.full-bundle/
- exchange messages
- navigate across messages

Screen reader only  includes the message but not the content of the status (30 minutes ago by example)

### What do you expect?

all information should be announced by Screen Reader while navigating in chat section. 

### What actually happened?

Only the message  is part of the screen reader narration

### MAS reference

https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html

### Adaptive Card JSON

```json

```
